pages/1.10/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -389,6 +390,9 @@ Indicates whether the master DNS port is open. An open master DNS port listens p
 *  `master_dns_bindall: 'true'` The master DNS port is open. This is the default value.
 *  `master_dns_bindall: 'false'` The master DNS port is closed.
 
+### master_external_loadbalancer
+The DNS name or IP address for the load balancer. If specified, this name is included in [DC/OS certificates](/1.10/security/ent/tls-ssl/).
```

Comment:
> this name is included

"this name" is weird with the previous "The DNS name or IP address".

Maybe "this is included as subject alternative name..."?

> included in [DC/OS certificates]

In all of them? Should we be more precise?

Reply:
Added a bit more specific info.